documentation/FAQ.md

```diff
@@ -0,0 +1,39 @@
+# FAQ in Swagger PR Review
+
+This page is intended to answer questions frequently asked during Azure Swagger PR review.
+
+1. [I am new to Swagger/OpenAPI, How should I start?](#Onboard)
+2. [How to fix validation failure?](#validation)
+3. [How to generate SDK from Swagger?](#SDK)
+4. [How to generate document](#doc)
+5. [If need further help, who can we contact?](#contact)
+
+## How to onboard PR review Process?<a name="Onboard"></a>
+If you are new to Swagger/OpenAPI, you can refer to this [document](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/80/Getting-started-with-OpenAPI-specifications)
```

Comment:
Wonder if this document is better? https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/71/AME-onboard-home